lib/utils/compiler.js

```diff
@@ -186,13 +186,12 @@ async function webpackInstance(opts) {
         throw new Error(processingErrors);
     }
 
-    const compiler = getCompiler(opts);
+    const compiler = getCompiler(options);
     let lastHash = null;
 
-    const ProgressPlugin = webpack.ProgressPlugin;
```

Comment:
Reason for doing so is mutation on several runs, I think it's fine, but if a regression comes up, this is the reason. We have to find an exact match, not instance match